requirements-sdk.in

```diff
@@ -7,4 +7,5 @@
 # Also avoid putting any libraries that are only used by the build system (directly or
 # indirectly).  Those belong in "requirements-build.in".
 websockets
-aiohttp
+aiohttp>=3.10.2
```

Comment:
In general, I don't think this kind of version specification is necessary.  Any run of `generateRequirements` will either leave this alone or pick a version that is greater since that's what would be available next.  There is no way for it to revert to an earlier version.  Doesn't hurt, but can be a bit confusing (IMO).

Reply:
I tried just doing the `pip install --upgrade` for it, but it didn't select the higher version.  Maybe something else is amiss with my installation, but that was my experience.

Reply:
You should use the "generateRequirements" task which uses "pip-compile" (from a suite called "piptools") to compute the full dependencies.  Don't run `pip` directly (at least not in any of the repos that have the `.in` files, which I think is all of them now).

Reply:
did that, still no joy.  I'll look into it more...

Reply:
Note that you have to use the `-Pupgrade=true` for it to look for "newer" versions of existing modules.  Otherwise, it will only pull those in if some newly added requirements needs a newer version to work (the advantage of "pip-compile" is that it does a full transitive dependency search and comes up with the total set needed to fulfill the named requirements which are the ones in the ".in" files).